Week2/homework/maartjes_work.js

```diff
@@ -45,3 +45,14 @@ const tuesday = [
 const tasks = monday.concat(tuesday);
 
 // Add your code here
+//console.log(tasks);
+let tasksHours = tasks.map((task)  => task.duration/60);
+//console.log(tasksHours);
+let tasksHoursfiltered = tasksHours.filter((taskhour) => taskhour >= 2 );
+//console.log(tasksHoursfiltered);
+let tasksPrice =tasksHoursfiltered.map((taskfiltered)  => taskfiltered*10);
+//console.log(tasksPrice);
+let salary = tasksPrice.reduce((x, taskPrice) => x + taskPrice, 0);
```

Comment:
Could you please rename `x` to something more meaningful like `total` or `sum`? It is quite useful to understand code with such names. 🙂 

Also, using reduce this way works perfectly fine but I would like to share another thing with you. If you don't specify an initial value of the `accumulator` which is `x` in your case, it will be automatically assigned the first element of the array and `taskPrice` would then point to the second element of the array. 

For example, if the array is like `[10,20,30,40]` and when we write reduce like you have done above, `x` will start with value `0` and `taskPrice` would be `10`. However, if you don't specify the `0 at the end, then `x` will start with value `10` and `taskPrice` would be `20` and your result would be the same in both cases. You can try it later.